docs/running-on-kubernetes.md

```diff
@@ -240,6 +240,38 @@ the command may then look like the following:
 
 ## Advanced
 
+### Running in-cluster client mode applications
+
+While Spark on Kubernetes does not officially support client mode applications, such as the PySpark shell, there is a workaround that
```

Comment:
Suggest rephrasing as:

"While Spark on Kubernetes does not support client mode applications, such as the PySpark shell, when launched from outside Kubernetes, Spark on Kubernetes does support client mode applications launched within the cluster.  This _in-cluster_..."

Reply:
Agreed.